gdx/src/com/badlogic/gdx/graphics/Color.java

```diff
@@ -326,9 +326,13 @@ public int toIntBits () {
 	public String toString () {
 		String value = Integer
 			.toHexString(((int)(255 * r) << 24) | ((int)(255 * g) << 16) | ((int)(255 * b) << 8) | ((int)(255 * a)));
-		while (value.length() < 8)
-			value = "0" + value;
-		return value;
+
+		StringBuilder stringBuilder = new StringBuilder();
+		for(int i = 0,len=8-value.length();i<len;++i) {
+			stringBuilder.append('0');
+		}
+		stringBuilder.append(value);
+		return stringBuilder.toString();
```

Comment:
`toString` is used for debugging, and since the code is longer and less readable, I'd remove this.